.github/workflows/prune.yml

```diff
@@ -0,0 +1,47 @@
+name: Prune GHCR
+
+on:
+  schedule:
+    - cron: '0 0 * * *'
+
+permissions:
+  contents: write
+  packages: write
+  security-events: write
+
+jobs:
+  prune_images:
+    name: Prune old sparrow images
+    runs-on: ubuntu-latest
+
+    permissions:
+      contents: write
+      packages: write
+      security-events: write
+
+    steps:
+    
+      - name: Prune Images
+        uses: vlaurin/action-ghcr-prune@v0.5.0
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          organization: caas-team
+          container: sparrow
+          dry-run: true
+          keep-younger-than: 7 # days
+          prune-untagged: true
+          prune-tags-regexes: |
+            ^commit-
+            SNAPSHOT-.*$
```

Comment:
After the first prune we can remove this regex because our new images don't follow this naming pattern anymore.

Reply:
The regex is unnecessary. We just want to remove old images, no matter what naming convention they used. Let's just remove it altogether.

Reply:
Then we'd also remove our stable images because git tags != image tags

Reply:
Totally right, my bad 😅